src/file.toit

```diff
@@ -251,9 +259,80 @@ Deletes a file, given its name.
 delete name/string -> none:
   #primitive.file.unlink
 
-/*
+/**
 Renames a file or directory.
 Only works if the $to name is on the same filesystem.
 */
 rename from/string to/string -> none:
   #primitive.file.rename
+
+/**
+Creates a hard link from $source to a $target file.
+*/
+link --hard --source/string --target/string -> none:
+  if not hard: throw "INVALID_ARGUMENT"
+  link_ source target LINK_TYPE_HARD_
+
+/**
+Creates a soft link from $source to a $target file.
+*/
+link --file --source/string --target/string -> none:
+  if not file: throw "INVALID_ARGUMENT"
```

Comment:
"Invalid argument" is too hard to debug.
```suggestion
  if not file: throw "SOURCE NOT A FILE"
```

Reply:
This is an attempt to prevent `--file=false`, so the proposed error message makes no sense.